templates/treat_ad.yaml

```diff
@@ -0,0 +1,61 @@
+Structural_Biology_Core_Folder: &struct_bio_folder
+  name: Structural_Biology_Core
+  type: Folder
+  next_level:
+    - Antibody_Validation_Data
+    - Assay_Protocols
+    - Assay_Results
+    - Cell_Lines
+    - Chemotypes
+    - Crystal_Structures
+    - Expression_Constructs
+    - Expression_Data
+    - Fragment_Screening_Results
+    - Lab_Notebooks
+    - Protein
+    - Protein_Expression_Methods
+    - TEP
```

Comment:
I'm assuming these are Folders?  By default are those the folder names if `name` isn't specified?

Reply:
Yes, it defaults to folder - since this is the structure to create projects.